packages/astro/src/@types/astro.ts

```diff
@@ -586,6 +586,23 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		server?: string;
+		/**
+		 * @docs
+		 * @name build.assets
+		 * @type {string}
+		 * @default `'_astro'`
```

Comment:
Do we also want to add `outDir` as a “see also” here given the final location is `${outDir}/${build.assets}`? 

```suggestion
		 * @default `'_astro'`
		 * @see outDir
```